src/core/router/router.blueprint.tsx

```diff
@@ -147,13 +147,20 @@ export const blueprint: RouteObject[] = [
           {
             path: 'list',
             loader: async () => {
-              const data = await jobs({ page: 1, status: 'ACTIVE', limit: 5 });
-              return data;
+              const currentIdentity = await identities();
+              const activeIdentity = currentIdentity.find((identity) => identity.current);
+              const requests = [
+                jobs({ identity_id: activeIdentity?.id, page: 1, status: 'ACTIVE' }),
+                jobs({ identity_id: activeIdentity?.id, page: 1, status: 'EXPIRE' }),
+                jobs({ page: 1, status: 'ACTIVE', limit: 5 }),
+              ];
+              const [activeJobs, archivedJobs, userJobs] = await Promise.all(requests);
+              return { activeJobs, archivedJobs, userJobs };
             },
             async lazy() {
               const { JobsList } = await import('src/Nowruz/pages/jobs/List');
               return {
-                Component: JobsList,
+                Component: Protect(JobsList),
```

Comment:
Jobs page is accessible even if user is not logged in, So it shouldn't be protected